quinn-proto/src/connection/mod.rs

```diff
@@ -131,8 +131,8 @@ where
     /// The "real" local IP address which was was used to receive the initial packet.
     /// This is only populated for the server case, and if known
     local_ip: Option<IpAddr>,
-
-    path: PathData,
+    /// public so that path data can be exposed to stats
+    pub path: PathData,
```

Comment:
This shouldn't be public. What does "exposed to stats" mean? I don't see any interfaces that would consume this.

Reply:
it's public so that I can do `self.0.lock("congestion").inner.path.congestion.clone_box()` in congestion_state()

Reply:
I've refactored it to make this not public.